lib/rules/no-classic-components.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const ERROR_MESSAGE =
+  'Use Glimmer components(@glimmer/component) instead of classic components(@ember/component)';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Enforces Glimmer components',
+      category: 'Best Practices',
```

Comment:
Could we change this category to `Ember Octane` please? We've been adding that to the other Octane rules.

Reply:
Sure thing 😄